fml/platform/posix/file_posix.cc

```diff
@@ -210,4 +211,20 @@ bool WriteAtomically(const fml::UniqueFD& base_directory,
                     base_directory.get(), file_name) == 0;
 }
 
+std::vector<std::string> ListFiles(const fml::UniqueFD& directory) {
+  std::vector<std::string> files;
+  DIR* dir = ::fdopendir(directory.get());
+  if (dir == nullptr) {
+    FML_LOG(ERROR) << "Can't open the directory.";
```

Comment:
Nit: Also log strerror(errno) so it is easier to tell why the directory could not be opened. Also, maybe this should be a debug log?